src/Pos/Wallet/Web/Server/Full.hs

```diff
@@ -1,31 +1,43 @@
+{-# LANGUAGE DataKinds           #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeOperators       #-}
 
 -- | Module for full-node implementation of Daedalus API
 
 module Pos.Wallet.Web.Server.Full
        ( walletServeWebFull
+       , walletServeWebFullS
        , walletServerOuts
+       , liftWMode
+       , runWStatsMode
+       , runWProductionMode
+       , runWStaticMode
+       , WalletStatsMode
+       , WalletProductionMode
+       , WalletStaticMode
        ) where
 
-import           Universum
```

Comment:
Wee, when someone once said we have to place `Universum` separatelly, I thought is was a joke. Till now I placed it exactly like here